### PR TITLE
Optimize backup-cleanup process

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/RestoreAutoBackup.cs [hunk 1]
```diff
@@ -166,30 +168,26 @@ private void linkLabelOpenContainingFolder_LinkClicked(object sender, LinkLabelL
...
                     {
-                        var name = Path.GetFileName(fileName);
-                        if (RegexFileNamePattern.IsMatch(name) && Convert.ToDateTime(name.Substring(0, 10), CultureInfo.InvariantCulture) < targetDate)
+                        try
                         {
-                            File.Delete(fileName);
-                            filesDeleted++;
-                            if (filesDeleted > maxCount)
+                            var name = Path.GetFileName(fileName);
+                            if (RegexFileNamePattern.IsMatch(name) && Convert.ToDateTime(name.Substring(0, 10), CultureInfo.InvariantCulture) <= targetDate)
```
Comment (ivandrofly) [inline src/Forms/RestoreAutoBackup.cs]: Here we want to delete backups after a specific month:
`Convert.ToDateTime(name.Substring(0, 10), CultureInfo.InvariantCulture) <= targetDate`

The old code was `Convert.ToDateTime(name.Substring(0, 10), CultureInfo.InvariantCulture) == targetDate` which I think is wrong...

File: src/Forms/RestoreAutoBackup.cs [hunk 2]
```diff
@@ -166,30 +168,26 @@ private void linkLabelOpenContainingFolder_LinkClicked(object sender, LinkLabelL
 
         public static void CleanAutoBackupFolder(string autoBackupFolder, int autoBackupDeleteAfterMonths)
         {
-            const int maxCount = 100; // to avoid locking computer
```
Comment (ivandrofly) [inline src/Forms/RestoreAutoBackup.cs]: Not needed 'cause we are running on a worker thread.

File: src/Forms/RestoreAutoBackup.cs (hunk 1 above)
Comment (niksedk) [reply src/Forms/RestoreAutoBackup.cs]: The old code was ´<´... (not `==`)

Comment (ivandrofly) [reply src/Forms/RestoreAutoBackup.cs]: > The old code was ´<´... (not `==`)
Exactly, typo

**Pull request merged**